### ISSUE TITLE
Among the Champion

### ISSUE THREAD

**Issue by humapal** [dalaranwow/dalaran-wow#4693]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Does not award a Mark  of the Champion each time you best an opponent.

**Expected behaviour**: Tell us what should happen instead
Each time you best a Champion you should receive a Mark of the Champion

**Steps to reproduce the problem**:

1. Become a Champion
2. Fight in the Champion Ring
3. Best an opponenet

**Include proofs for this behaviour**
![champion](https://cloud.githubusercontent.com/assets/26784243/25603330/70626c02-2ec0-11e7-9b0b-b29dfc18b3dd.jpg)

You can see that I bested 6 different factions  in 4 minutes and 40 seconds and in the Quest tracker I only received 2 marks of the champion.  BTW, that needs to be an achievement in and of itself.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

http://www.wowhead.com/quest=13790/among-the-champions

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @humapal , the champions share the same problem as the valiants and will be adressed in #4692
Thank you.
